frontend/src/app/main/ui/context.cljs

```diff
@@ -13,6 +13,7 @@
 
 (def embed-ctx (mf/create-context false))
 (def render-ctx (mf/create-context nil))
+(def def-ctx (mf/create-context false))
```

Comment:
I think this context need some explanation note because it is not obvious the purpose of this ctx on the first/quick look on the code.

Reply:
It's explained in the place where it's used. 
https://github.com/penpot/penpot/pull/443/files#diff-a1591439ff86a209f08e8db2535263b0405739726587a1dc03fe4b5d6416d8b1R81

```clojure
          ;; We cannot wrap inside groups the shapes that go inside the defs tag
          ;; we use the context so we know when we should not render the container
```